### PR TITLE
https://issues.redhat.com/browse/ACM-6737 Add nutanix 2.9 front

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @oafischer. Since this the changes in this front-port is in your section, please review and see if it looks ok to fit in with the updates that you made in 2.9. Thanks!

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc [hunk 1]
```diff
@@ -1,9 +1,7 @@
 [#creating-a-cluster-on-premises]
 = Creating a cluster in an on-premises environment
 
-You can use the console to create an on-premises {ocp} cluster.
-
-You can use this procedure to create {sno} clusters, multi-node clusters, and compact three-node clusters on VMware vSphere, Red Hat OpenStack, Red Hat Virtualization Platform, and in a bare metal environment.
+You can use the console to create on-premises {ocp} clusters. The clusters can be single-node OpenShift (SNO) clusters, multi-node clusters, and compact three-node clusters on VMware vSphere, Red Hat OpenStack, Red Hat Virtualization Platform, Nutanix, or in a bare metal environment.
```
Comment (oafischer) [inline clusters/cluster_lifecycle/create_cluster_on_prem.adoc]: Is there a reason for not using {sno}?

**Review by oafischer — APPROVED** [review]
One comment for consideration, otherwise looks great. Thanks!

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc (hunk 1 above)
Comment (cadawson) [reply clusters/cluster_lifecycle/create_cluster_on_prem.adoc]: Nope. I changed it. Thanks!

**Pull request merged**